tests/Cache/CacheClientTest.php

```diff
@@ -2799,6 +2858,129 @@ public function testSetAddElementsSetFetch_RefreshTtl()
         $this->assertEquals($elements, $fetchedElements);
     }
 
+    public function testSetContainsElements_ExactMatch()
+    {
+        $setName = uniqid();
+        $elements = ["foo", "bar", "baz"];
+
+        $response = $this->client->setAddElements($this->TEST_CACHE_NAME, $setName, ["foo", "bar", "baz"], CollectionTtl::fromCacheTtl()->withNoRefreshTtlOnUpdates());
+        $this->assertNull($response->asError());
+        $this->assertNotNull($response->asSuccess(), "Expected a success but got: $response");
+
+        $response = $this->client->setContainsElements($this->TEST_CACHE_NAME, $setName, $elements);
+        $this->assertNull($response->asError());
+        $this->assertNotNull($response->asHit(), "Expected a hit but got: $response");
+
+        $expectedExists = [true, true, true];
+        $expectedExistsDict = [];
+        $expectedExistsDict["foo"] = true;
+        $expectedExistsDict["bar"] = true;
+        $expectedExistsDict["baz"] = true;
+
+        $this->assertEquals($response->asHit()->exists(), $expectedExists);
+        $this->assertEquals($response->asHit()->existsDictionary(), $expectedExistsDict);
+    }
+
+    public function testSetContainsElements_SubsetOfExistingSet()
+    {
+        $setName = uniqid();
+        $elements = ["foo", "bar", "baz"];
+
+        $response = $this->client->setAddElements($this->TEST_CACHE_NAME, $setName, ["foo", "bar", "baz", "bam", "beep", "boop"], CollectionTtl::fromCacheTtl()->withNoRefreshTtlOnUpdates());
+        $this->assertNull($response->asError());
+        $this->assertNotNull($response->asSuccess(), "Expected a success but got: $response");
+
+        $response = $this->client->setContainsElements($this->TEST_CACHE_NAME, $setName, $elements);
+        $this->assertNull($response->asError());
+        $this->assertNotNull($response->asHit(), "Expected a hit but got: $response");
+
+        $expectedExists = [true, true, true];
```

Comment:
Shouldn't this be: [true, true, true, false, false, false]?

Reply:
no, my request only passed in `$elements`, which has 3 items in it.